src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreMvcBuilderExtensions.cs

```diff
@@ -59,6 +60,56 @@ public static class MvcCoreMvcBuilderExtensions
         }
 
         /// <summary>
+        /// Adds an <see cref="ApplicationPart"/> to the list of <see cref="ApplicationPartManager.ApplicationParts"/> on the
+        /// <see cref="IMvcBuilder.PartManager"/>.
+        /// </summary>
+        /// <param name="builder">The <see cref="IMvcBuilder"/>.</param>
+        /// <param name="part">The <see cref="ApplicationPart"/>.</param>
+        /// <returns>The <see cref="IMvcBuilder"/>.</returns>
+        public static IMvcBuilder AddApplicationPart(this IMvcBuilder builder, ApplicationPart part)
```

Comment:
I suggest instead of this `AddAssemblyPart` - and call the other one `ConfigureParts`